src/app/templates/app/submit_new_license.html

```diff
@@ -352,12 +352,15 @@
               data: form,
               success: function (data) {
                 var githubCode = data.statusCode;
+                var githubIssueId = data.issueId;
+                var githubUrl = "https://github.com/spdx/licnse-list-xml/issues/";
```

Comment:
Note that there is a typo here resulting in a 404 on redirect: `licnse`

Reply:
Thanks @whot for pointing out the error, I will crate a PR to solve this